src/app/projects/components/create-new-attribute/create-new-attribute.component.html

```diff
@@ -263,9 +263,73 @@
 
 
             <kern-logs [logs]="attributeLogs"></kern-logs>
+
+            <div class="mt-8">
+                <div class="text-sm leading-5 font-medium text-gray-700 inline-block">Calculation progress</div>
+                <div *ngIf="currentAttribute.progress == 0 || currentAttribute.state == 'INITIAL'" class="bg-white">
+                    <div class="py-6 text-sm leading-5 font-normal text-gray-500">This attribute was not yet run.</div>
+                </div>
+                <div *ngIf="currentAttribute.progress > 0 && currentAttribute.progress < 1"
```

Comment:
i think we should also display on progress  = 0 (but state running or something like that). Since the progress bar now "blinks in" after some calculation is already run